spark3/src/main/java/org/apache/iceberg/spark/procedures/ExpireSnapshotsProcedure.java

```diff
@@ -77,12 +76,11 @@ public StructType outputType() {
 
   @Override
   public InternalRow[] call(InternalRow args) {
-    String namespace = args.getString(0);
-    String tableName = args.getString(1);
-    Long olderThanMillis = args.isNullAt(2) ? null : DateTimeUtils.toMillis(args.getLong(2));
-    Integer retainLastNum = args.isNullAt(3) ? null : args.getInt(3);
+    String tableName = args.getString(0);
```

Comment:
I think we better call our variables `tableIdent` rather than `tableName` now. I think we can keep the parameter name as `table`.

Reply:
sounds good, renames all around